examples/generic/connect.hpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018, Niklas Hauser
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef MODM_EXAMPLE_GENERIC_CONNECT_HPP
+#define MODM_EXAMPLE_GENERIC_CONNECT_HPP
+
+/* This is a helper file that maps the connectivity of the generic examples
+ * to the specific board layout.
+ */
+
+namespace Generic
+{
+
+#ifdef GENERIC_EXAMPLE_UART
+
+namespace Uart
+{
+
+using Tx = GpioOutputA9;
+using Instance = Usart1;
+
+void
+connect()
+{
+	Instance::connect<Tx::Tx>();
+    Instance::initialize<Board::systemClock, 115200>();
+}
+
+}
+
+#endif // GENERIC_EXAMPLE_LOGGER
+
+#ifdef GENERIC_EXAMPLE_CAN
+namespace Can
+{
+
+#if defined(MODM_BOARD_NUCLEO_144)
+using Tx = Board::D66;
+using Rx = Board::D67;
+#elif defined(MODM_BOARD_NUCLEO_64)
+using Tx = Board::D14;
+using Rx = Board::D15;
+#else
+using Tx = GpioOutputB9;
+using Rx = GpioInputB8;
+#endif
+
+// using Tx2 = GpioOutputB13;
```

Comment:
?

Reply:
Adding Can2 loopback mode or something. I'm not sure anymore how much of this is example and how much is an integration test. 😒 hwut?

Reply:
Ok